extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_linkAuthentication.php

```diff
@@ -4,6 +4,7 @@
 <a href="<?= Sanitizer::encodeAttribute( $href ); ?>"
    rel="nofollow"
    id="<?= Sanitizer::encodeAttribute( $model['title']['key'] ); ?>"
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "title" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)